Assignments/Program.cs

```diff
@@ -11,34 +11,39 @@
 using System.Linq;
 using System.IO;
 using System.Diagnostics;
+using System.Reflection;
 
 namespace Academy;
 public class A14 {
    public static void Main () {
-      var wordList = new List<string> (File.ReadAllLines (@"c:/etc/words.txt"));
+      // Loading all the dictionary words from a assembly-manifest resource file.
+      using var reader = new StreamReader (Assembly.GetExecutingAssembly ().GetManifestResourceStream ($"Assignments.Files.words.txt"));

```

Comment:
You could have used File.ReadAllLines like taught in class. Why do you want to embed this file with project assembly.

Reply:
I thought making the file as an embedded resource will make the code faster.